shell/platform/common/flutter_platform_node_delegate.h

```diff
@@ -129,11 +129,17 @@ class FlutterPlatformNodeDelegate : public ui::AXPlatformNodeDelegateBase {
   ///             Subclasses must call super.
   virtual void Init(std::weak_ptr<OwnerBridge> bridge, ui::AXNode* node);
 
- protected:
   //------------------------------------------------------------------------------
-  /// @brief      Gets the underlying ax node for this accessibility node.
+  /// @brief      Gets the underlying ax node for this platform node delegate.
   ui::AXNode* GetAXNode() const;
 
+  //------------------------------------------------------------------------------
+  /// @brief      Gets the owner of this platform node delegate. This is useful
+  ///             when you want to get the information surround this platform
+  ///             node delegate, e.g. the global rect of this platform node
+  ///             delegate. This pointer is only safe in the platform thread.
```

Comment:
Since this is only safe to use on the Platform thread, we should probably use a `fml::WeakPtr`, which will actually do assertions around usage of this, rather than `std::weak_ptr` which is meant to be thread safe.

Reply:
We're close to being able to use it, but not quite. Can you add a TODO here?